docs/intermittent_producer.md

```diff
@@ -0,0 +1,21 @@
+## What is intermittent streaming?
+Intermittent streaming is when you want to send and stop video at will.  For example consider a doorbell camera which only wants to record motion events.  The application might want to start producing frames as soon as it detects motion and then stop at some point when there is no longer motion and then not stream again until it detects motion. 
```

Comment:
>Intermittent streaming is when you want to send and stop video at will

Should clarify that sending and stopping video at will *without tearing and setting up SDK pipeline again*